guides/v2.1/cloud/bk-cloud.md

```diff
@@ -1,24 +1,29 @@
 ---
 group: cloud
-title: Welcome to Magento Commerce (Cloud)
+title: Magento Commerce Cloud guide
 functional_areas:
   - Cloud
 ---
 
-{{site.data.var.ece}}, formerly Enterprise Cloud Edition, is a managed and automated hosting platform for Magento specifically created for Cloud infrastructures. This version combines {{site.data.var.ee}}, Cloud infrastructure hosting, with a few differences and added features, including Git integration and key environments for development, staging, and live production. You can code, test, and deploy across Integration, Staging, and Production environments for _continuous integration_ of your store.
+{{site.data.var.ece}} is a managed, automated hosting platform for the {{site.data.var.ee}} software. {{site.data.var.ece}} comes with a variety of additional features that sets it apart from the on-prem {{site.data.var.ee}} and {{site.data.var.ce}} platforms:
 
-Each {{site.data.var.ece}} plan includes an Integration environment for development, testing, and integrating services. Additional active environments are available at additional cost.
+![Cloud Benefits]
 
-Cloud plans include:
+{{site.data.var.ece}} includes features such as git-based **Rapid development** with a pre-provisioned infrastructure that includes PHP, MySQL, Redis, RabbitMQ, and Elasticsearch technologies; **Continuous deployment** every time you push code changes in a PaaS environment; highly **Customizable**  configuration files and tools; and AWS hosting that offers merchants a **Scalable** and **Secure** environment for online sales and retailing.
 
--   Active Git branches of code.
--   Unlimited number of *inactive* Git branches.
--   Robust {% glossarytooltip d85e2d0a-221f-4d03-aa43-0cda9f50809e %}Composer{% endglossarytooltip %} build process.
--   And the option to add fully managed services like MySQL, Elasticsearch, Redis, RabbitMQ, and so on, without requiring external add-ons.
+## Technology stack
 
-The {{site.data.var.ece}} architectures use the following technologies:
+Amazon Web Services (AWS) powers the underlying Infrastructure as a Service for {{site.data.var.ece}}. Each {{site.data.var.ece}} plan provides a PaaS Integration environment for developing, testing, and integrating services, see [{{site.data.var.ece}} architecture]({{page.baseurl}}/cloud/architecture/cloud-architecture.html) for a comparison and additional features. Performance tools provide granular visibility into the site performance and PHP profiling. [Fastly]({{page.baseurl}}/cloud/cdn/cloud-fastly.html) caches site assets and, as customers access the site and stores, loads the cached assets fast. Also, the Fastly Web Application Firewall (WAF) prevents malicious traffic and other OWASP Top 10 threats from affecting the site.
```

Comment:
Maybe make the parenthetical more explicit using parentheses: "...and integrating services (see Magento Commerce Cloud architecture for...)".

Also, consider moving the details of the tech stack below the graphic. What I've found to work well is to write a brief intro before the graphic--one or two sentences max--to describe the graphic at 1000 ft. Something to prime readers for what they're seeing at avoid putting important details in a place they may skip by jumping right the the graphic. Something like this: "One way to understand the Magento Commerce Cloud is to think of it as five functional layers, as shown here:" Not this: "The following graphic shows an overview of the technology stack", which is already provided by the heading. In general, people typically jump to the graphic and start looking at it, then read further to understand what they're seeing. Highlighting the terms in the paragraph to match the words/phrasing of the graphic is also nice, like you did for the first graphic.

Reply:
Got it. Will move stuff around.